opensrp-eusm/src/main/java/org/smartregister/eusm/config/AppTaskingLibraryConfiguration.java

```diff
@@ -95,9 +95,8 @@ public Float getLocationBuffer() {
     public void startImmediateSync() {
         LocationTaskServiceJob.scheduleJobImmediately(LocationTaskServiceJob.TAG);
         SyncServiceJob.scheduleJobImmediately(SyncServiceJob.TAG);
-        SyncStockTypeServiceJob.scheduleJobImmediately(SyncStockTypeServiceJob.TAG);
         ImageUploadServiceJob.scheduleJobImmediately(ImageUploadServiceJob.TAG);
-        SyncStockServiceJob.scheduleJobImmediately(SyncStockServiceJob.TAG);
+        //SyncStockServiceJob.scheduleJobImmediately(SyncStockServiceJob.TAG);
```

Comment:
Why is this removed?

Reply:
It should be called when the structures so as to populate the `locations` filter have been fetched so i moved the call to `AppLocationIntentService`